execution_workflows/Aptardi/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use an official Python runtime as a parent image
+FROM nfcore/base:1.13
+
+# Install any needed packages specified in requirements.txt
+#RUN apt-get update && apt-get install -y --no-install-recommends build-essential samtools
+#RUN apt-get install bedtools
+#RUN apt-get update && apt-get install -y wget && rm -rf /var/lib/apt/lists/*
+RUN conda create -n aptardi --channel conda-forge -c bioconda aptardi samtools bedtools && conda clean -a
+
+# Add conda installation dir to PATH (instead of doing 'conda activate')
+ENV PATH /opt/conda/envs/aptardi/bin:$PATH
+
+# Dump the details of the installed packages to a file for posterity
+RUN conda env export --name aptardi > aptardi.yml
+
+# Copy the current directory contents into the container at /
```

Comment:
Again not super familiar with best practices for Docker. Doesn't this run the risk of different package versions being installed if the file is pulled/created at different times (e.g. in 1 month if bedtools/samtools is updated and this breaks aptardi)? Would it be better to specify package versions in line 8?

This comment in the README is a little concerning... https://github.com/luskry/aptardi#requirements

@uniqueg any reservations about using conda inside our Dockerfiles?

Reply:
One can change the version when building a new container `docker build account/tag:version`, so `apaeval:aptardi:1.0` will not be overwritten on docker hub. I tested out `apt install bedtools:version and samtools:version` in the Docker (normally this works with most other softwares), but `aptardi` didn't recognized the installation.